package.json

```diff
@@ -15,13 +15,13 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/e-e-e/random-access-s3.git"
+    "url": "git+https://github.com/random-access-storage/random-access-s3.git"
   },
   "dependencies": {
-    "abstract-random-access": "^1.1.2",
-    "inherits": "^2.0.3"
+    "random-access-storage": "^1.1.0"
   },
   "devDependencies": {
+    "aws-sdk": "^2.188.0",
```

Comment:
this is a prod dep right?

Reply:
ah yes it is! thank you for catching that one